lib/SIL/Dominance.cpp

```diff
@@ -14,22 +14,22 @@
 #include "swift/SIL/SILBasicBlock.h"
 #include "swift/SIL/SILArgument.h"
 #include "swift/SIL/Dominance.h"
-#include "llvm/Support/GenericDomTree.h"
 #include "llvm/Support/GenericDomTreeConstruction.h"
 
 using namespace swift;
 
 template class llvm::DominatorTreeBase<SILBasicBlock, false>;
 template class llvm::DominatorTreeBase<SILBasicBlock, true>;
 template class llvm::DomTreeNodeBase<SILBasicBlock>;
-using SILDomTree = llvm::DomTreeBase<SILBasicBlock>;
-using SILPostDomTree = llvm::PostDomTreeBase<SILBasicBlock>;
-template void
-llvm::DomTreeBuilder::Calculate<SILDomTree, swift::SILFunction>(
+
+namespace llvm {
+namespace DomTreeBuilder {
+template void Calculate<SILDomTree, swift::SILFunction>(
     SILDomTree &DT, swift::SILFunction &F);
-template void
-llvm::DomTreeBuilder::Calculate<SILPostDomTree, swift::SILFunction>(
+template void Calculate<SILPostDomTree, swift::SILFunction>(
     SILPostDomTree &DT, swift::SILFunction &F);
+} // namespace DomTreeBuilder
+} // namespace llvm
```

Comment:
You ought to be able to do the same thing as the lines above:

```
  template void llvm::DomTreeBuilder::Calculate
```

but it's not a big deal.

Reply:
The SILDomTree and SILPostDomTree types are in that same namespace, so I thought it was more clear this way without having to repeat that namespace prefix for the "DT" argument types.